### DISCUSSION TITLE
How to edit svg style in tailwindcss typography?

### DISCUSSION THREAD

**Discussion by LeenHawk** [tailwindlabs/tailwindcss#12021] [Help] [answered]
There is only `prose-img` label, not svg

**Comment by wongjn** [comment, marked as answer]
You could consider using an [arbitrary variant](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants) like `[&_svg]:class-name`.

**Reply by LeenHawk** [reply]
Thank you
